.github/workflows/test_and_deploy.yml

```diff
@@ -71,8 +58,7 @@ jobs:
           yarn prettier:test
 
       - name: Test
-        run: |
-          yarn test-ci
+        run: yarn test-ci
 
       - name: Check docs links
```

Comment:
This step requires a build of the docs. Removing the build means that the check isn't done.